packages/react/src/primitives/utils/useLayoutEffect.ts

```diff
@@ -8,8 +8,6 @@ import * as React from 'react';
  *
  * See: https://reactjs.org/docs/hooks-reference.html#uselayouteffect
  */
-const useLayoutEffect = Boolean(globalThis?.document)
-  ? React.useLayoutEffect
-  : () => {};
+const useLayoutEffect = globalThis?.document ? React.useLayoutEffect : () => {};
```

Comment:
Is there a linting rule against using Boolean?

Reply:
No just redundant usage ([see this comment](https://github.com/aws-amplify/amplify-ui/pull/2334#discussion_r927973321))